serverless/knative_eventing/serverless-channels.adoc

```diff
@@ -0,0 +1,29 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-channels"]
+= Using Channels
+include::modules/common-attributes.adoc[]
+:context: serverless-channels
+
+toc::[]
+
+It is possible to sink events from an event source to a Knative Eventing channel.
+Channels are Kubernetes Custom Resources (CRs) which define a single event forwarding and persistence layer.
+After events have been sent to a channel, these events can be sent to multiple Knative services by using a subscription.
+
+The default configuration for channel instances is defined in the `default-ch-webhook` ConfigMap, however developers can still create their own channels directly by instantiating a supported channel object.
+
+== Supported channel types
+Currently, {ServerlessProductName} only supports the use of InMemoryChannel channels as part of the Knative Eventing Technology Preview.
+
+[IMPORTANT]
+====
+InMemoryChannel channels are a development only channel and are not suitable for use in production.
+====
+
+The following are limitations of InMemoryChannel channels:
+
+* No event persistence is available. If a pod is goes down, events on that pod will be lost.
```

Comment:
s/If a pod is goes down/If a Pod goes down/
s/that pod/that Pod/
_Use initial capitalization and camel case for Kubernetes/OpenShift API objects_
https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#api-object-formatting